keyboards/ungodly/nines/readme.md

```diff
@@ -0,0 +1,16 @@
+# Nines
+
+![Nines](https://i.imgur.com/zNnmPV9.jpg)
+
+A 9 key (3x3) macropad with rotary encoder support.
+Made by Ungodly Design for Qlavier.
+
+* Keyboard Maintainer: [Luis Godinez](https://github.com/luis-Godinez)
+* Hardware Supported: The Nines PCB
+* Hardware Availability: [Qlavier](https://www.qlavier.com/shop/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ungodly/nines:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?